engines/engine.h

```diff
@@ -587,9 +587,16 @@ class Engine {
 	inline Common::SaveFileManager *getSaveFileManager() { return _saveFileMan; }
 
 public:
-	/** On some systems, check whether the game appears to be run from CD. */
-	void checkCD();
-
+	/** Check if extraced CD Audio files are found.
+	*/
+	bool existExtractedCDAudioFiles();
+	/** On some systems, check whether the game appears to be run
+	* from the same CD drive, which also should play CD audio.
+	*/
```

Comment:
Same here, concerning multi-line comments

Reply:
Fixed in next commit: https://github.com/scummvm/scummvm/pull/3018/commits/f76acd499e81a06ec603e3beed8e97114d30230e